modules/apps/user/user-test/src/testIntegration/java/com/liferay/user/service/test/UserSetDigestTest.java

```diff
@@ -64,19 +64,16 @@ public void testAddUserWithWorkflowWithPrerequisites() throws Exception {
 	}
 
 	@Test
-	public void testSetDigestAfterPrerequisites() throws Exception {
-		User user = _userLocalService.createUser(RandomTestUtil.nextLong());
-
-		user.setScreenName(RandomTestUtil.randomString());
-		user.setEmailAddress(_generateRandomEmailAddress());
-
-		String digest = user.getDigest(RandomTestUtil.randomString());
-
-		Assert.assertNotNull(digest);
-
-		user.setDigest(digest);
+	public void testDigestIsEmptyAfterCreatingUser() throws Exception {
+		User user = _testAddUserWithWorkflowHelper(
+			RandomTestUtil.randomString(), _generateRandomEmailAddress());
 
-		Assert.assertEquals(digest, user.getDigest());
+		Assert.assertEquals(
+			"User digest should be empty after user creation", user.getDigest(),
+			"");
```

Comment:
If we want to company only empty string here then we should use `StringPool.BLANK`